examples/apps/hugging_face_integration_app/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM nvcr.io/nvidia/clara-holoscan/holoscan:v2.1.0-dgpu
```

Comment:
Can we use monai-deploy CLI packager instead, and providing the requirements file and the app config file?

Reply:
ya we can try that. I will have a look tonight.